packages/flutter_tools/bin/fuchsia_tester.dart

```diff
@@ -115,13 +111,22 @@ Future<Null> run(List<String> args) async {
       collector = new CoverageCollector();
     }
 
+
+    Map<String, String> precompiledDillFiles = {};
+    List<Map<String, dynamic>> json = List<Map<String, dynamic>>.from(
+      jsonDecode(
+        fs.file(argResults[_kOptionPrecompiledDillFiles]).readAsStringSync()));
+    for (Map<String, dynamic> map in json) {
+      precompiledDillFiles[map["source"]] = map["dill"];
```

Comment:
Why not just have the raw JSON content be the desired map?

```
{
  "foo": "bar",
  "baz": "qux"
}
```

Reply:
That's what I started with, but then had difficulty generating such json using gn write_file()